manim_slides/present.py

```diff
@@ -727,6 +734,9 @@ def update_canvas(self) -> None:
         self.display_width, self.display_height = self.thread.current_resolution
         if not self.isFullScreen():
             self.resize(self.display_width, self.display_height)
+        self.label.setStyleSheet(
+            f"background-color: {self.thread.current_presentation.background_color}"
+        )
```

Comment:
```suggestion
        self.label.setStyleSheet(
            f"background-color: {self.thread.current_background_color}"
        )
```

For readability, create a `current_background_color` property :-)